modules/districtmDMXBidAdapter.js

```diff
@@ -103,6 +103,10 @@ export const spec = {
       dmxRequest.source.ext.schain = schain || {}
     } catch (e) {}
     let tosendtags = bidRequest.map(dmx => {
+      if (dmx.params.coppa) {
```

Comment:
Instead of using your config parameter, please use the global one.  See PR #5012 for an example.

Reply:
thanks for the example